include/tc/core/mapping_options.h

```diff
@@ -295,6 +295,7 @@ class SchedulerOptionsView {
   /// Construct a view that refers to a protocol buffers message.
   SchedulerOptionsView(const SchedulerOptionsView&) = default;
   SchedulerOptionsView(SchedulerOptionsProto& buf) : proto(buf) {}
+  SchedulerOptionsView(SchedulerOptionsProto&& buf) : proto(buf) {}
```

Comment:
This does not seem like a good idea independently.
We may want to do it nonetheless in the big picture but atm I would punt on adding such extensions.